src/skeleplex/app/_utils.py

```diff
@@ -1,15 +1,18 @@
 from IPython import get_ipython
 from qtpy.QtCore import QTimer
 from qtpy.QtWidgets import QApplication
-
+from magicgui import magicgui
+import sys
 from skeleplex.app import DataManager, SkelePlexApp, SkeletonDataPaths
-
+from skeleplex.app._curate import make_split_edge_widget, ChangeBranchColorWidget
+import networkx as nx
 # store reference to QApplication to prevent garbage collection
 _app_ref: QApplication | None = None
 
 
 def view_skeleton(
     graph_path: str,
+    launch_widgets: bool = True,
```

Comment:
I think this is fine for now. Eventually, I would like to be able to set the viewer into different "modes" (e.g., curate). each mode would load the relevant widgets and setup the viewer state. In any case, that's something for a future PR, just mentioning where we are headed.